app/javascript/src/NewService/components/FormElements/ServiceDiscoveryListItems.jsx

```diff
@@ -1,28 +1,56 @@
 // @flow
 
-import React from 'react'
+import React, {useState, useEffect} from 'react'
 import {Label, Select} from 'NewService/components/FormElements'
+import {fetchData} from 'utilities/utils'
+import {BASE_PATH} from 'NewService'
+
+import type { ClusterResource } from 'NewService/types'
 
 type Props = {
-  fetchServices: (namespace: string) => Promise<void>,
-  projects: string[],
-  services: string[]
+  projects: ClusterResource[],
```

Comment:
Seems a bit odd to have a type for something that holds just a prop that is a string. Can't we just return the name from Rails and have a type `string[]`

Reply:
I understand what you say, I decided to create a type so that it is extensible. Maybe eventually we require other fields like a human name. But it you think that's not likely going to happen I agree it is enough with strings.

Reply:
Let's go for just strings to make it obvious and same-same with services. Also, regarding my previous comment in https://github.com/3scale/porta/pull/1015#discussion_r308259340, we could just return the names (strings) array

Reply:
Ah, I see what you meant now